app/views/not_subscribed_to_cds.scala.html

```diff
@@ -29,32 +27,20 @@
 
 @layout(pageTitle = Some(messages("cf.not-subscribed-to-cds.detail.title"))) {
     @h1(messages("cf.not-subscribed-to-cds.detail.heading"))
-    @h2(messages("cf.not-subscribed-to-cds.detail.how-to-get-access-info"))
+    @h2(messages("cf.not-subscribed-to-cds.detail.already-subscribed-to-cds"))
+
     <p class="govuk-body">
-        @messages("cf.not-subscribed-to-cds.detail.eori-info")
-        @hmrcNewTabLink(
-            NewTabLink(
-                language = Some(messages.lang.toString),
-                classList = Some("govuk-link"),
-                href=Some(appConfig.registerCdsUrl),
-                text=messages("cf.not-subscribed-to-cds.detail.eori-link")
-            )
-        )
-        .
+        @messages("cf.not-subscribed-to-cds.detail.already-subscribed-to-cds-guidance-text")
     </p>
 
+    @h2(messages("cf.not-subscribed-to-cds.details.subscribe-to-cds"))
+
     <p class="govuk-body">
```

Comment:
Will probably need to add this class `govuk-!-margin-bottom-7` to the P tag for extra spacing below